vrm_samples/load_at_runtime_scene.gd

```diff
@@ -0,0 +1,9 @@
+extends Spatial
+
+func _ready():
+	var vrm_loader = load("res://addons/vrm/vrm_loader.gd").new()
+	
+	var model: Spatial = vrm_loader.import_scene("res://vrm_samples/AliciaSolid_vrm-0.51.vrm", 1, 1000)
```

Comment:
Why does this load alicia?

Reply:
Alicia is provided as an example model already. In order to illustrate that models can be loaded via code at runtime, the Alicia model is loaded.